base/sort.jl

```diff
@@ -1067,14 +1067,16 @@ end
 ## sortperm: the permutation to sort an array ##
 
 """
-    sortperm(v; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
+    sortperm(A; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward, [dims::Integer])
 
-Return a permutation vector `I` that puts `v[I]` in sorted order. The order is specified
+Return a permutation vector or array `I` that puts `A[I]` in sorted order along the given dimension.
+If `A` is an `AbstractArray`, then the `dims` keyword argument must be specified. The order is specified
```

Comment:
```suggestion
If `A` has more than one dimension, then the `dims` keyword argument must be specified. The order is specified
```
Because `Vector <: AbstractArray`